index.html

```diff
@@ -1,16 +1,120 @@
+
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link href='https://fonts.googleapis.com/css?family=Raleway' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+    <title>Cute Cakes</title>
 </head>
-
 <body>
-    <!-- Add your markup here -->
+    <main class="homepage-main">
+        <section class="homepage-hero">
+            <header>
+                <svg viewBox="0 0 100 80" width="40" height="40">
+                    <rect width="100" height="20"></rect>
+                    <rect y="30" width="100" height="20"></rect>
+                    <rect y="60" width="100" height="20"></rect>
+                </svg>
+                <img class="image" src="http://www.clker.com/cliparts/9/5/G/c/a/4/pink-frosted-cupcake-md.png" width="200px" alt="massive cake">
+                <h4>Cute Cakes</h4>
```

Comment:
Hey Ika, I think you need to keep headings in order (H1,H2, etc). I can see you used a 'H4' before a 'H3'. Avoid using headings for design elements. Use CSS for style rather than using a heading level that doesn’t fit the hierarchy. Well done for your effort.

Reply:
Hi Amarachi, thank you for the review. I have made changes to my code.